service/gcs/core/gcs/cleanup.go

```diff
@@ -20,7 +20,7 @@ func (c *gcsCore) cleanupContainer(containerEntry *containerCacheEntry) error {
 	}
 
 	diskMap := containerEntry.MappedVirtualDisks
-	disks := make([]prot.MappedVirtualDisk, 0, len(diskMap))
```

Comment:
Correct me if I'm wrong, but if we use make like this, doesn't it create a slice of length `len(diskMap)` full of default elements, and then `append` just adds on to this slice? So we'd end up with a slice half full of default `MappedVirtualDisk`s?

Reply:
You could also do it by not specifying the capacity as your have, and assigning the disk/dir at the correct index in the assignment loop as well. I honestly like that better I think :)

Reply:
Oh good call.